### DISCUSSION TITLE
Call for submissions: Community Driven Solutions For Multiplexed Tissue Imaging Using The IBEX Method

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#155] [Announcements]
Interested in sharing well-documented protocols that extend the utility of the original IBEX method? Consider submitting your protocols to [the upcoming methods collection](https://app.jove.com/methods-collections/3182) of The Journal of Visualized Experiments ([JoVE](https://app.jove.com/)).

## An image is worth a thousand words, a video is worth so much more

[Why publish in JoVE](https://app.jove.com/authors/overview):
* Increase reproducibility of your protocol.
* Retain lab knowledge and efficiently train new members and collaborators.

[The process](https://app.jove.com/authors/publication):

* Manuscript submission.
* Editorial review.
* Peer review.
* If accepted, film and video production (one of three options):
  1. JoVE’s team handles the script writing, videography, and editing (institution is within JoVE's videographer network).
  2. JoVE's team handles script writing and editing, you are responsible for videography.
  3. You handle everything  and submit it to JoVE for approval.

Ready to submit your work? Go to the [IBEX collection page](https://app.jove.com/methods-collections/3182).

**Comment by radtkea** [comment]
Awesome! Thank you so much for showcasing this opportunity Zivy!

Abstracts: July 16th, 2024
Full length articles: Late September 2024

Reach out to me or Dr. Koenig if you have questions!
